.github/workflows/reusable_build_maturin_linux.yml

```diff
@@ -56,11 +65,39 @@ jobs:
           maturin-version: latest
           command: build
           manylinux: 2014
-          args: --find-interpreter --out wheels -m ${{inputs.py_interface_folder}}/Cargo.toml --release --locked
+          args: --out wheels -m ${{inputs.py_interface_folder}}/Cargo.toml --release --locked -i python3.9 python3.10 python3.11 python3.12
+      - name: store artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: linux_wheels_p_3.9_3.10_3.11_3.12
+          path: wheels
+
+  build_maturin_builds_linux_py313:
+    if: ${{ inputs.python_3_13 }}
+    name: maturin_build-linux
+    runs-on: 'ubuntu-latest'
+    steps:
+      - uses: actions/checkout@v4
+      # - uses: Swatinem/rust-cache@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          default: true
```

Comment:
Or is this then overwritten here?

Reply:
No that's just for `actions-rs/toolchain@v1` https://github.com/actions-rs/toolchain?tab=readme-ov-file#inputs